instana/singletons.py

```diff
@@ -94,11 +94,10 @@ def set_agent(new_agent):
 # this package.
 tracer = InstanaTracer(recorder=span_recorder)
 
-if sys.version_info >= (3, 4):
+if sys.version_info >= (3, 7):
```

Comment:
Not necessarily the topic of this PR, but this whole if statement should be removed.

Reply:
True. I will change it.